### PR TITLE
Initial build for 3.1.4 ❄️

### PR DESCRIPTION
**Pull request by JeanChristopheMorinPerso**
Upstream: https://github.com/magmax/python-inquirer/tree/v3.1.4

https://anaconda.atlassian.net/browse/PKG-3560

Channel: Snowflake

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,20 +12,19 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38]
```
Comment (boldorider4) [inline recipe/meta.yaml:15]: maybe add a comment why that is

**Review by boldorider4 — APPROVED** [review]
LGTM but minor comment

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:15]: These statements are very common (in pretty much all feedstocks) and it's almost always the same reason: upstream doesn't support the skipped python versions.

**Review by danpetry — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:15]: Indeed, see https://inspector.pypi.io/project/inquirer/3.1.4/packages/3c/b7/2b4a352659d78386e0f7f5405486aa1c41664666d33e9e8506640f702f70/inquirer-3.1.4-py3-none-any.whl/inquirer-3.1.4.dist-info/METADATA#line.9.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**